.gitignore

```diff
@@ -14,6 +14,7 @@ node_modules/
 .env.development.local
 .env.test.local
 .env.production.local
+.sample.env
```

Comment:
We wouldn't want to ignore this file ever - if other developers were to get onboarded it's easier to show what environment variables they need to run the app. 

It's just important not to expose any credentials in that file

Reply:
I have reverted and removed the keys but kept the file.